server/lib/models/persona.go

```diff
@@ -10,6 +10,19 @@ type Persona struct {
 }
 
 func InitPersona() {
-	create_table(Persona{}, "personas", "Id")
+	t := create_table(Persona{}, "personas", "Id")
+	t.AddIndex("account_channel", "INDEX", []string{"Account", "Channel"})
 }
 
+func FindPersona(dbif Dbif, account, channel proto.UUID) (*Persona, error) {
+	p := &Persona{
+		proto.Model_Persona {
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] too many values in struct literal